CalibFormats/HcalObjects/src/HcalText2DetIdConverter.cc

```diff
@@ -206,7 +207,8 @@ bool HcalText2DetIdConverter::init (const std::string& fFlavor, const std::strin
   else if (flavorName.find ("ZDC_") == 0) {
     HcalZDCDetId::Section section = flavorName == "ZDC_EM" ? HcalZDCDetId::EM :
       flavorName == "ZDC_HAD" ? HcalZDCDetId::HAD : 
-      flavorName == "ZDC_LUM" ? HcalZDCDetId::LUM : HcalZDCDetId::Unknown;
+      flavorName == "ZDC_LUM" ? HcalZDCDetId::LUM : 
+      flavorName == "ZDC_RPD" ? HcalZDCDetId::RPD : HcalZDCDetId::Unknown;
```

Comment:
implement this with a switch or regular "if - else if" to improve readability.
``` a = c1 ? r1: c2? r2: c3? r3 : c4 ? r4 : r5;``` is not easily readable.